src/app/featureSets/basic/render2D.js

```diff
@@ -756,5 +756,344 @@ export const render2D = {
         featurePrimitiveType: "blah",
         targetPrimitiveType: "blah",
         targetPrimitiveSet: "Basic2D"
-    }
+    },
+
+    //new
+    Filter: {
+        featureParams: {
+            position: "position",
+            orientation: "orientation",
```

Comment:
Remove the orientation, and put rotation. We are going to change all the components to use rotation instead of rotation